Update pr template (#2439)
chaos-mesh/chaos-mesh: .github/pull_request_template.md

<!--
Thank you for contributing to Chaos Mesh!

> If you're not sure how to do it, please read the CONTRIBUTING document of Chaos Mesh first.

Reviewer: ```suggestion
If you're unsure where to start, please refer to the contributing doc:
```

Reviewer: Rest LGTM

https://github.com/chaos-mesh/chaos-mesh/blob/master/CONTRIBUTING.md

If you still have questions, please let us know via issues.

Please follow the Title Formats below when you open a new PR:

1. module[, module2, module3]: what's changed
2. *: what's changed
-->

### What problem does this PR solve?

<!-- Uncomment this line if some issues to close -->

> <!-- Close #xxx -->

Reviewer 2: I think `#<issue number>` is better.

Author: Sounds good. 💯 I will update it soon.

Author: Done in dd040e5.
